### PR TITLE
#279 - Initializing the cypress and preparing the first smoke test

### PR DESCRIPTION
**Pull request by sebastian-vass**
- Initialization of cypress
- Preparation of the first smoke test

### CODE REVIEW

File: next/frontend/utils/formState.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { GenericObjectType, retrieveSchema, RJSFSchema } from '@rjsf/utils'
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (sebastian-vass) [inline next/frontend/utils/formState.ts:2]: I ran into this problem with the current eslint project setup. I didn't want to install unnecessary packages or add a rule to eslint, so I threw it into ignor. What to do with it?

File: next/tsconfig.json [hunk 2]
```diff
@@ -30,5 +30,5 @@
     }
   },
   "include": ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx", "next-env.d.ts"],
-  "exclude": ["node_modules", "schema-generator"]
+  "exclude": ["node_modules", "schema-generator", "./cypress.config.ts", "cypress"]
```
Comment (sebastian-vass) [inline next/tsconfig.json:33]: Cypress causing type errors in Jest assertions. So there was a problem with failing the typescript check.

File: next/frontend/utils/formState.ts (hunk 1 above)
Comment (mpinter) [reply next/frontend/utils/formState.ts:2]: The library not being in package is an issue, feel free to add it. You can do it in a separate PR, bude I don't mind doing it in here as well.

File: next/tsconfig.json (hunk 2 above)
Comment (mpinter) [reply next/tsconfig.json:33]: This does not look right, if you exclude the entire cypress directory and the tsconfig inside of it extends this config, then I don't think the files would be correctly processed by the editor ? At least my local VS Code complains in every file I open form the cypress directory:

Screenshot 2023-11-15 at 15 42 32

**Pull request closed without merging**